modules/carto/src/api/layer-map.ts

```diff
@@ -74,6 +78,15 @@ const AGGREGATION_FUNC = {
   variance
 };
 
+const DOCUMENT_LAYER_TYPE_TO_LAYER: Record<TileLayerType, ConstructorOf<Layer>> = {
```

Comment:
Should be `TILE_LAYER_TYPE_TO_LAYER`

Reply:
Wops, so many things at the same time, sorry 😢